otter/test/convergence/test_service.py

```diff
@@ -104,24 +109,31 @@ def test_buckets_acquired(self):
         When buckets are allocated, the result of converge_all_groups is
         performed.
         """
-        def converge_all_groups(log, currently_converging, _my_buckets,
+        def converge_all_groups(currently_converging, _my_buckets,
                                 all_buckets, divergent_flags):
             return Effect(
-                ('converge-all', log, currently_converging, _my_buckets,
+                ('converge-all', currently_converging, _my_buckets,
                  all_buckets, divergent_flags))
 
         my_buckets = [0, 5]
-        sequence = SequenceDispatcher([
-            (GetChildren(CONVERGENCE_DIRTY_DIR), lambda i: ['flag1', 'flag2']),
-
+        bound_sequence = [
```

Comment:
I think this looks better, but one of my original points was specifically to include these intents in-line in the main sequence list, so the order is nicely represented lexically in the code.  Just a stylistic thing, though :)